articles/azure/azs-how-link-storage-explorer.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to link Azure Storage Explorer to Azure Stack | UKCloud Ltd
+description: Provides help for linking Azure Storage Explorer to Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to link Azure Storage Explorer to Azure Stack
+toc_fullpath: Users/How To/Create a virtual machine/azs-how-link-storage-explorer.md
+toc_mdlink: azs-how-link-storage-explorer.md
+---
+
+# How to link Azure Storage Explorer to Azure Stack
+
+In this guide, you will learn how to link Azure Stack subscriptions to the Azure storage explorer. Azure storage explorer is a standalone application that allows you to easily work with Azure Stack storage data on Windows, macOS and Linux. This tool allows for easy downloading and uploading to Azure Stack which makes tasks such as downloading and uploading data disks to VM's very easy.
+
+If you need to install the storage explorer, please find the link [here](https://azure.microsoft.com/en-us/features/storage-explorer/) and install it.
```

Comment:
If you need to install Storage Explorer, you can find the link ...

Reply:
fixed